lighthouse-plugin-publisher-ads/audits/loads-gpt-from-sgdn.js

```diff
@@ -63,7 +63,7 @@ class LoadsGptFromSgdn extends Audit {
     if (!gptUrl) {
       return auditNotApplicable.NoGpt;
     }
-    const passed = (gptUrl.host === 'securepubads.g.doubleclick.net');
+    const passed = (gptUrl.host === 'securepubads.g.doubleclick.net' || gptUrl.host === 'cdn.ampproject.org');
```

Comment:
This change is no longer needed? GPT is not hosted on cdn.ampproject.org

Reply:
It is the AMP ad script that is hosted on the cdn.ampproject domain. without this change I was getting a message at the and of the process "No GPT script was found". Maybe that is expected? I was trying to remove that error because it is confusing to see on AMP sites.

To make this clearer I could create a separate `LoadsAMPAdsFromAMPCDN` helper and check both here?

Reply:
completed this.